### PR TITLE
Introduce Critical issue type (add label automatically)

### PR DESCRIPTION
**Pull request by julienbourdeau**

### CODE REVIEW

File: tests/AppBundle/webhook_examples/pull_request_body.critical.json [hunk 1]
```diff
@@ -0,0 +1,412 @@
...
+        "repos_url": "https://api.github.com/users/weaverryan/repos",
+        "events_url": "https://api.github.com/users/weaverryan/events{/privacy}",
+        "received_events_url": "https://api.github.com/users/weaverryan/received_events",
+        "type": "User",
+        "site_admin": false
+      },
+      "repo": {
+        "id": 21151556,
+        "name": "symfony",
+        "full_name": "weaverryan/symfony",
```
Comment (mickaelandrieu) [inline tests/AppBundle/webhook_examples/pull_request_body.critical.json]: look at others exemples, it's not the good repository name

File: tests/AppBundle/webhook_examples/pull_request_body.critical.json [hunk 2]
```diff
@@ -0,0 +1,412 @@
...
```
Comment (mickaelandrieu) [inline tests/AppBundle/webhook_examples/pull_request_body.critical.json]: Same here

Comment (mickaelandrieu) [inline tests/AppBundle/webhook_examples/pull_request_body.critical.json]: Same here

Comment (mickaelandrieu) [inline tests/AppBundle/webhook_examples/pull_request_body.critical.json]: Same here

Comment (mickaelandrieu) [inline tests/AppBundle/webhook_examples/pull_request_body.critical.json]: Same here

Comment (mickaelandrieu) [inline tests/AppBundle/webhook_examples/pull_request_body.critical.json]: Same here

Comment (mickaelandrieu) [inline tests/AppBundle/webhook_examples/pull_request_body.critical.json]: Same here

Comment (mickaelandrieu) [inline tests/AppBundle/webhook_examples/pull_request_body.critical.json]: Same here

Comment (mickaelandrieu) [inline tests/AppBundle/webhook_examples/pull_request_body.critical.json]: Same here

File: tests/AppBundle/webhook_examples/pull_request_body.critical.json [hunk 3]
```diff
@@ -0,0 +1,412 @@
...
+    "commits": 110,
+    "additions": 4476,
+    "deletions": 1393,
+    "changed_files": 181
+  },
+  "repository": {
+    "id": 21151556,
+    "name": "symfony",
+    "full_name": "loveOSS/test",
+    "owner": {
```
Comment (mickaelandrieu) [inline tests/AppBundle/webhook_examples/pull_request_body.critical.json]: This is the correct full_name to use in tests :)

File: src/AppBundle/EventSubscriber/PullRequestSubscriber.php [hunk 4]
```diff
@@ -137,6 +138,21 @@ public function checkForNewTranslations(GitHubEvent $githubEvent)
...
+    public function checkIfPrFixCriticalIssue(GitHubEvent $githubEvent)
+    {
+        $labelWasAdded = $this->container
+            ->get('app.issue_listener')
+            ->addLabelCriticalLabelIfNeeded($githubEvent->getEvent()->pullRequest)
+        ;
+
+        if ($labelWasAdded) {
+            $githubEvent->addStatus([
+                'event' => 'pr_edited',
```
Comment (mickaelandrieu) [inline src/AppBundle/EventSubscriber/PullRequestSubscriber.php]: Regarding your event declaration, it should be 'pr_created'

File: tests/AppBundle/webhook_examples/pull_request_body.critical.json (hunk 3 above)
Comment (julienbourdeau) [inline tests/AppBundle/webhook_examples/pull_request_body.critical.json]: Merci :)

**Pull request merged**